sdks/python/apache_beam/io/fileio_test.py

```diff
@@ -188,7 +188,7 @@ def test_read_entire_file_bzip2_empty(self):
     self.assertEqual(read_lines, [])
 
   def test_read_entire_file_bzip2_large(self):
-    lines = ['Line %d' % d for d in range(10 * 1000)]
+    lines = ['Line %d' % d for d in range(100 * 1000)]
```

Comment:
The sum of all additional tests and changes in this file after this PR seems to add ~10 seconds.
(From 15 seconds before to 25 seconds after).